MAINTAINING.md

```diff
@@ -96,3 +96,401 @@ write effective API documentation, such as setting up the RST files and docstrin
+
+
+Traditionally, once Tier 1 is live on PyPI, the post-release actions in step 5 can start, including the announcements.
+The rest of the tiers might take even longer and they also need to be approved.
+
+
+### 5. Post-release actions
+
+#### 5.1 Announce the release on Slack
+
+Post a message in the relevant Slack channels:
+
+ * IBM internal channels: all the releases.
+ * [Qiskit organization](https://qiskit.enterprise.slack.com/)
+    - Roadmap announcement `#roadmap-announcements`: Especially release candidates. All other releases can be posted in the thread of the release candidate announcement.
+    - General channel `#general` and Qiskit developer `#qiskit-dev`: All the stable major and minor releases.
+
+Examples for announcements:
+
+For a release candidate:
+> :qiskit-new: Qiskit x.y.zrc1 is now live on Github (link) and PyPI(link)! :rocket:
+>
+> As this is a pre-release pip will not install it automatically, you will have to manually specify the version with: `pip install "qiskit==x.y.zrc1"`
+> The x.y.z final release is planned for X weeks from now. If you encounter any issues with the release candidate, please [file an issue](https://github.com/Qiskit/qiskit/issues/new/choose) so we can address them before the final release.
+
+For minor releases:
+> :qiskit-new: **Qiskit x.y is out!**
+>
+> * Fully backwards compatible with x.0. As always, following [Semantic Versioning](https://qisk.it/semver)
+> * A technical release summary will be published on [the IBM blog](https://www.ibm.com/quantum/blog) in about a week
+>
+> or
+>
+> * Here is technical release summary (link to the blog post)
+> * Take a look to the release notes (link)
+> Don't forget that the Qiskit vX series has bug fixing support until XXth, XXXX and security support until YYth, YYYY.
+
+For patch releases:
+> :qiskit-new: **Qiskit x.y.z has been released!**
+> This is a minor bugfix release for Qiskit x.y. You can find it on pypi (link) and in our GitHub releases (link).
+
+#### 5.2 Update the `main` branch with the next release
+
+> [!IMPORTANT]
+> Skip this step if your PR from section Releasing the package - step 3 was to the  `main` branch
+
+Make a PR to the `main` branch that sets the version number to the _next_ minor (e.g. `2.2.0` if you have just released `2.1.0rc1`).
+The places to update are:
+
+* `qiskit/VERSION.txt`: the only line in the file.
+* `docs/conf.py`: the variables `version` and `release`.
+* `Cargo.toml` (only the file in the repository root and none of the other `**/Cargo.toml`s): the variable `version` and run `cargo build`.
+* `crates/cext/cbindgen.toml`: the `#define QISKIT_VERSION_*` macros.
+* `.mergify.yml`: change the `backport` action to target the new stable branch that `qiskit-bot` created as part of the release (see [*Backporting* section](#Backporting) for details on Mergify).
+
+This opens the `main` branch for feature development for the next release.
+
+Examples for post-release PR:
```

Comment:
In general these are not the best examples since they don't follow the version labeling method introduced in #14697. That said, we still need an example here so I suggest we keep just one example and add the caveat that the versions should follow the convention introduced in #14697 and not as in the example.